src/Payload/Level.php

```diff
@@ -2,6 +2,17 @@
 
 class Level implements \JsonSerializable
 {
+    const IGNORE = 'ignore';
```

Comment:
What are these two "ignore(d)" levels for? They don't map to any level we have on the backend. Same for "alert" and "emergency".